frontend/packages/console-app/src/components/quick-starts/data/quick-start-data.ts

```diff
@@ -2,10 +2,11 @@ import { exploreServerlessQuickStart } from './explore-serverless-quickstart';
 import { explorePipelinesQuickStart } from './explore-pipeline-quickstart';
 import { installAssociatePipelineQuickStart } from './install-associate-pipeline-quickstart';
 import { serverlessApplicationQuickStart } from './serverless-application-quickstart';
+import { QuickStart } from '../utils/quick-start-types';
 
 export const allQuickStarts = [
   exploreServerlessQuickStart,
   serverlessApplicationQuickStart,
   explorePipelinesQuickStart,
   installAssociatePipelineQuickStart,
-];
+] as QuickStart[];
```

Comment:
Please add the type information to the variable instead of 'casting' your array. This will ensure that your types really matches. Try this by adding for a example a string to your array.

```diff
-export const allQuickStarts = [
+export const allQuickStarts: QuickStart[] = [
  exploreServerlessQuickStart,
  serverlessApplicationQuickStart,
  explorePipelinesQuickStart,
  installAssociatePipelineQuickStart,
-] as QuickStart[];
+];
```

This will require that you add `: Quickstart` also to the quick starts json definitions itself, but only then we use the types here to validate the data.

Reply:
Done